### ISSUE TITLE
Incorporate changes from GSI tag gfsda.v16.3.5

### ISSUE THREAD

**Issue by EdwardSafford-NOAA** [NOAA-EMC/GSI-Monitor#66]
In the latest operational version of the GSI (tag gfsda.v16.3.5) the DA monitors are still included in the GSI repository.  Evaluate the differences between the DA monitor components in the gfsda.v16.3.5 tag and the GSI-monitor repository and incorporate all those appropriate to the GSI-monitor repo.

Note to self -- need to update the RadMon base file for avhrr_metop-c, update and rerun OznMon MkBase script.

**Comment by EdwardSafford-NOAA** [comment]
Tested changes using the latest global-workflow/develop branch in an abbreviated run, running only the gdasvrfy job.  The RadMon, MinMon, and OznMon each ran to completion without errors and produced the expected output.

I needed to update the base (historic values) files for both the RadMon and OznMon, and found I'd not yet updated either of the MkBase scripts.  I went ahead and made those changes here, though it's unrelated to the gfsda.v16.3.5 changes.
